File: runtime/android/app/src/main/java/com/mobvoi/WeTextProcessing/WeTextProcessing.java
package com.mobvoi.WeTextProcessing;

public class WeTextProcessing {

  static {
    System.loadLibrary("wetextprocessing");
  }

  public static native void init(String modelDir);
  public static native String normalize(String input);
  public static native String inverse_normalize(String input);
}

File: runtime/android/app/src/main/java/com/wenet/WeTextProcessing/WeTextProcessing.kt
package com.wenet.WeTextProcessing

object WeTextProcessing {

    init {
        System.loadLibrary("wetextprocessing")
    }

    @JvmStatic
    external fun init(modelDir: String)

    @JvmStatic
    external fun normalize(input: String): String

    @JvmStatic
    external fun inverse_normalize(input: String): String
}
